Day 2/TipCalculator.py

```diff
@@ -0,0 +1,14 @@
+print("Welcome to the tip calculator")
+bill = input("What was the total bill?  ")
+percentage = input("What percentage tip would you like to give ? 10, 12, or 15? ")
+people = input("How many people to split the bill? ")
+
```

Comment:
To indicate each time the data type in expressions, it is better to immediately specify the type at the input.
`sult_percentage = int(percentage)/100*int(bill)`
`bill = int(input("What was the total bill?  "))`
then you will have:
`sult_percentage = percentage / 100* bill)` in other places to